app/scripts/models/reliers/oauth.js

```diff
@@ -160,6 +180,8 @@ define([
           // server version always takes precedent over the search parameter
           self.set('redirectUri', serviceInfo.redirect_uri);
           self.set('origin', Url.getOrigin(serviceInfo.redirect_uri));
+          // parse resume token to set additional properties for the relier
+          self._parseResumeToken();
```

Comment:
Should resume token values override values from the server? If resume token values are like search params, the server values should take precedent.